test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6429,10 +6429,7 @@ def manager(self, cmd, obj_type, attrib=None, id=None, extend=None,
         issue a management command to the server, e.g to set an
         attribute
 
-        Returns the return code of ``qmgr/pbs_manager()`` on
-        success, if expect is set to True, the return value is
-        that of the call to expect.Raises ``PbsManagerError`` on
-        error
+        Returns 0 for Success and non 0 number for Failure
```

Comment:
instead of non-zero value for failure, manager() should also raise exception. Why are we changing that?

Reply:
Thanks for catching that Anamika, I got confused because I didn't realize that expect raises an exception as well, which means that manager() used to raise an exception even when expect was set to True. I'll fix it back.